### ISSUE TITLE
Out of range dot still shows on Bars 4-8

### ISSUE THREAD

**Issue by siven80** [tullamods/tullaRange#33]
The vanilla out of range dot still shows on Bars 4-8.

Bars 1-3 have it removed when addons is installed, but bars 4-8 still show the out of range dot.

**Comment by Tuller** [comment]
Do you have bindings assigned to those bars?

**Comment by siven80** [comment]
No, i have keybinds on bars 1-3, then have bar 6 showing up in combat with abilities on them instead of weakauras.

**Comment by siven80** [comment]
Also noticed bars 4-6 dont set the unusable color option sometimes. Noticed this when on warrior with execute being lit up all the time like a usable button.

Weird thing is sometimes it colors them properly, yet other times it doesnt.

only had tullarange addon installed.
